### DISCUSSION TITLE
Different heating values for steering wheel and rear window

### DISCUSSION THREAD

**Discussion by carpaij** [Hyundai-Kia-Connect/kia_uvo#279] [Show and tell]
First of all: awesome HA add on! I'm driving an IONIQ 5 and love how it now integrates within the HA eco-system. 

Today I started playing around with the kia_uvo.start_climate service and noticed that when I set heating to true both the steering wheel and the rear window heating turn on. In the original app I can enable them separately. I could not find any documentation around this, neither on bluelinky. So I started to play around a little....

I tried sending different values for heating instead of the default 'true' and that worked!
- value '0' no heating (only AC)
- value '1' and '4' both mean rear window and steering wheel
- value '2' means only rear window
- value '3' means only steering wheel

To be more precise: when the rear window is turned on the side mirror heating is turned on as well. This is the same as in the original app.

This might be interesting to other users.

**Comment by cdnninja** [comment]
This is awesome research!  Understanding what the car accepts is half the battle with these things.   Thanks for doing this.  Which brand and region are you using?

**Comment by nprez83** [comment]
Just wanted to share that I tried doing this with my Kia Telluride (US) and unfortunately it looks like the steering wheel and rear windows/side mirror are locked to each other. Specifically, I tried sending the start_climate command with '0' for rearWindow and sideMirror,  and '1' for  steeringWheel, and although the command was transmitted successfully, all three came on. I guess I'm not surprised since that's the way it works in the native Kia UVO app (a single setting for "heating accessories"), I was just hoping there was some extended functionality they had not enabled, looks like that's not the case. Thanks as always for the amazing work with this integration.
